### PR TITLE
Add defrag event check steps after do system disk shrink

### PR DESCRIPTION
**Pull request by peixiu**
Add defrag event check steps after do system disk shrink.

ID: 1200
Signed-off-by: Peixiu Hou <phou@redhat.com>

### CODE REVIEW

**Comment by peixiu** [comment]
This patch based on this patch: https://github.com/avocado-framework/avocado-vt/pull/3743

**Comment by peixiu** [comment]
python3 ConfigTest.py --testcase=block_resize.with_iozone.with_system_disk --guestname=Win11 --platform=x86_64 --clone=no --machines=q35 --driveformat=virtio_scsi --customsparams="cdrom_virtio = isos/windows/virtio-win-prewhql-0.1-240.iso\nnum_queues = 32" --vcpu=32

 (1/1) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.block_resize.with_iozone.with_system_disk.q35: STARTED
 (1/1) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.block_resize.with_iozone.with_system_disk.q35: PASS (697.25 s)

**Comment by peixiu** [comment]
@qingwangrh @YongxueHong Could you help to review this patch?

Thanks~
Peixiu

**Comment by qingwangrh** [comment]
python3 ConfigTest.py --testcase=block_resize.with_iozone.with_system_disk --guestname=Win2019 --platform=x86_64 --clone=no --machines=q35 --driveformat=virtio_scsi --firmware=default_bios --netdst=virbr0


(1/1) Host_RHEL.m9.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_resize.with_iozone.with_system_disk.q35: STARTED
 (1/1) Host_RHEL.m9.u3.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_resize.with_iozone.with_system_disk.q35: PASS (417.04 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

LGTM

**Comment by peixiu** [comment]
@YongxueHong Could you help to review this patch? thanks~

**Comment by peixiu** [comment]
python3 ConfigTest.py --testcase=block_resize.with_iozone.with_system_disk --guestname=Win10 --platform=x86_64 --clone=no --machines=q35 --driveformat=virtio_scsi --customsparams="cdrom_virtio = isos/windows/virtio-win-prewhql-0.1-240.iso\nnum_queues = 32" --vcpu=32

 (1/1) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.block_resize.with_iozone.with_system_disk.q35: STARTED
 (1/1) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.block_resize.with_iozone.with_system_disk.q35: PASS (368.86 s)

**Comment by peixiu** [comment]
@menli820 @YongxueHong Could you help to check this patch?

Thanks~
Peixiu

File: qemu/tests/cfg/block_resize.cfg [hunk 1]
```diff
@@ -14,6 +14,8 @@
         max_block_size = 2199023255552
         ovmf:
             max_block_size = 17592186044416
+        258_event_check = 'wevtutil qe Application /q:"*[System/EventID=258]" /f:text'
```
Comment (YongxueHong) [inline qemu/tests/cfg/block_resize.cfg:17]:
```suggestion
        check_258_event = 'wevtutil qe Application /q:"*[System/EventID=258]" /f:text'
```
Recommend the command parameter starts with the verb.

Comment (peixiu) [reply qemu/tests/cfg/block_resize.cfg]: Get and thanks! Updated done~

**Comment by menli820** [comment]
LGTM

 (1/2) Host_RHEL.m9.u3.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: STARTED
 (1/2) Host_RHEL.m9.u3.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (782.11 s)
 (2/2) Host_RHEL.m9.u3.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_resize.with_iozone.with_system_disk.q35: STARTED
 (2/2) Host_RHEL.m9.u3.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.block_resize.with_iozone.with_system_disk.q35: PASS (336.34 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Review by menli820 — APPROVED** [review]

**Pull request merged**